takwimu/models/dashboard.py

```diff
@@ -599,6 +606,45 @@ def __str__(self):
         return self.question.encode('ascii', 'ignore')
 
 
+class FeaturedAnalysisBlock(blocks.StructBlock):
+    country = blocks.ChoiceBlock(choices=country_choices)
+    featured_page = blocks.PageChooserBlock(target_model=ProfileSectionPage)
+    name = 'featured_analysis'
+
+
+class IndexPage(ModelMeta, Page):
+    """
+    The process for integrating Wagtail into an existing project is definitely
+    less polished than starting a project from Wagtail's built-in template.
+    The project template comes with an editable HomePage model.
+    When you add wagtail to an existing project, it automatically create a root
+    page for your site that can not be edited or extended
+    (https://github.com/wagtail/wagtail/issues/3992)
+    """
+    analysis = StreamField([
+        ('featured_analysis', FeaturedAnalysisBlock())
+    ], blank=True)
+
+    content_panels = Page.content_panels + [
+        StreamFieldPanel('analysis')
+    ]
+
+    def get_context(self, request, *args, **kwargs):
+        settings = CountryProfilesSetting.for_site(request.site)
+        published_status = settings.__dict__
+
+        context = super(IndexPage, self).get_context(request, *args, **kwargs)
+        context['explainer_steps'] = ExplainerSteps.objects.first()
+        context['faqs'] = FAQ.objects.all()
+        context['testimonials'] = Testimonial.objects.all().order_by('-id')[:3]
+
+        context.update(settings(request))
+        context.update(settings(request))
```

Comment:
The above two lines are repeated and they cause the following error on view:

![Screenshot from 2019-04-04 15-29-42](https://user-images.githubusercontent.com/1779590/55555742-a3186180-56ee-11e9-84b3-4bff7b86ba69.png)